lib/services/local/desktops.go

```diff
@@ -139,15 +131,41 @@ func (s *WindowsDesktopService) UpsertWindowsDesktop(ctx context.Context, deskto
 }
 
 // DeleteWindowsDesktop removes the specified windows desktop resource.
-func (s *WindowsDesktopService) DeleteWindowsDesktop(ctx context.Context, name string) error {
-	err := s.Delete(ctx, backend.Key(windowsDesktopsPrefix, name))
-	if err != nil {
-		if trace.IsNotFound(err) {
-			return trace.NotFound("windows desktop %q doesn't exist", name)
+func (s *WindowsDesktopService) DeleteWindowsDesktop(ctx context.Context, hostID, name string) error {
```

Comment:
What happens if `name` is empty too? Would it delete all?

I wonder if we should make that an error, since there's a separate RPC to delete all.